lib/fireblocks/jsonhttp.go

```diff
@@ -0,0 +1,98 @@
+package fireblocks
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"io"
+	"net/http"
+	"net/url"
+
+	"github.com/omni-network/omni/lib/errors"
+	"github.com/omni-network/omni/lib/log"
+)
+
+// jsonHTTP provides a simple interface for sending JSON HTTP requests.
+type jsonHTTP struct {
```

Comment:
renamed to jsonhttp since this is more specific that just httpclient. Also added comments

Reply:
nit: I'm still a fan of some sort of `client` in the name. The fact that it's serializing as json is not so important implementation detail and I wouldn't include it in the name.